src/client/app/chat/chat.component.ts

```diff
@@ -255,13 +260,33 @@ export class ChatComponent implements OnInit, AfterViewChecked {
     this.createForm();
     this.receiveMessageFromSocket();
     this.receiveUpdatedMessageFromSocket();
-    //this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+    this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+  }
+  
+  //Doctor modal window open methhode
+  openDoctor(doctorModal:any) {
+    this.getDoctors();
+    this.modalService.open(doctorModal, {size: 'lg'}).result.then((result) => {
+      this.closeResult = `Closed with: ${result}`;
+    }, (reason) => {
+      this.closeResult = `Dismissed ${this.getDismissReason(reason)}`;
+    });
+  }
+  
+  private getDismissReason(reason: any): string {
+    if (reason === ModalDismissReasons.ESC) {
+      return 'by pressing ESC';
+    } else if (reason === ModalDismissReasons.BACKDROP_CLICK) {
+      return 'by clicking on a backdrop';
+    } else {
+      return  `with: ${reason}`;
+    }
   }
 
   ngAfterViewChecked() {
     setTimeout(() => {
       let dropdown = this.dropdown.nativeElement;
-      if (this.selectedUser.role !== 'patient') {
+      if (this.selectedUser.role !== 'user') {
```

Comment:
There is no role called `user`. Kindly change it to `patient`, as it was before.